ThunderCatsTests/UICollectionReusableViewTests.m

```diff
@@ -0,0 +1,36 @@
+//
+//  UICollectionReusableViewTests.m
+//  ThunderCats
+//
+//  Created by Abbey Jackson on 2016-05-12.
+//  Copyright © 2016 Metova. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "UICollectionReusableView+TCAdditions.h"
+
+@interface UICollectionReusableViewTests : XCTestCase
+
+@end
+
+@implementation UICollectionReusableViewTests
+
+- (void)testReuseIdentifier
+{
+    NSString *identifier = [UICollectionReusableView tc_reuseIdentifier];
+    XCTAssertEqualObjects(identifier, @"UICollectionReusableView");
```

Comment:
I'm not sure how to do this in objC, do you know of a project that has this done as an example I can look at?